src/NuGet.Clients/NuGet.CommandLine/Commands/PackCommand.cs

```diff
@@ -98,6 +97,10 @@ public override void ExecuteCommand()
 
             Console.WriteLine(LocalizedResourceManager.GetString("PackageCommandAttemptingToBuildPackage"), Path.GetFileName(packArgs.Path));
 
+            // If the BasePath is not specified, use the current directory
```

Comment:
I already checked-in a change for this : https://github.com/NuGet/NuGet.Client/commit/f94068960ec5cb56c1dc518ad819d21ec0d77087